### DISCUSSION TITLE
Normalised count data

### DISCUSSION THREAD

**Discussion by Abbie-Williams** [FrederickHuangLin/ANCOMBC#152] [Q&A]
Hi, 

How do we get the normalised/corrected datatables back from the new ancombc2 function?

Thanks!

:)

**Comment by FrederickHuangLin** [comment]
Hi,

You can generate the bias-corrected abundances of `ancombc2` following the same method in `ancombc`. See "Bias-corrected abundances" section in this [vignette](https://www.bioconductor.org/packages/release/bioc/vignettes/ANCOMBC/inst/doc/ANCOMBC.html).

Best,
Huang
